Gemfile.lock

```diff
@@ -256,7 +256,7 @@ GEM
       rex-core
       rex-struct2
       rex-text
-    rex-core (0.1.11)
+    rex-core (0.1.12)
```

Comment:
yeah, all of the things are setup so that HEAD always is tagged as the next version, including metasploit-framework